sa/sa.go

```diff
@@ -643,20 +699,40 @@ func (ssa *SQLStorageAuthority) SetOrderError(ctx context.Context, req *sapb.Set
 		return nil, errIncompleteRequest
 	}
 	_, overallError := db.WithTransaction(ctx, ssa.dbMap, func(tx db.Executor) (interface{}, error) {
-		om, err := orderToModel(&corepb.Order{
-			Id:    req.Id,
-			Error: req.Error,
-		})
+		var omv1 *orderModelv1
+		var omv2 *orderModelv2
```

Comment:
This function doesn't need the omv1/omv2 treatment at all, because it's not touching the .Profile column in the database. It should be able to just use `orderToModelv2` right away.